### PR TITLE
Update toolltip / extended tooltip for VDE and modif contact

### PR DESCRIPTION
**Pull request by alcapat**
wait datenherr status

### CODE REVIEW

File: chsdi/templates/htmlpopup/verkehrszaehlstellen.mako [hunk 1]
```diff
@@ -92,44 +80,19 @@ display:none !important;
...
-<h1>Jahresauswertungen</h1>
- <tr>
-% for a in translist(c['attributes']['jahresauswertung']):
-   % if a[1] != None:
-       <a style="color:red;" href="${a[1]}" target="_blank">Jahresauswertung ${a[0]}</a>
-       <br/>
-   % endif
-% endfor
- </tr>
+<p>Um zus&auml;tzliche Informationen &uuml;ber Z&auml;hlstellen zu erhalten, kontaktieren Sie bitte <a href="mailto:verkehrsdaten@astra.admin.ch">verkehrsdaten@astra.admin.ch</a></p>
```
Comment (oterral) [inline chsdi/templates/htmlpopup/verkehrszaehlstellen.mako]: Not translated ?

**Comment by alcapat** [comment]
Not yet, we wait a translation (not urgent)

**Comment by oterral** [comment]
ok lgtm

File: chsdi/templates/htmlpopup/verkehrszaehlstellen.mako [hunk 2]
```diff
@@ -19,11 +8,10 @@ def translist (jahresauswert):
...
-    <tr><td class="cell-left">${_(nummer)}</td>             <td>${c['featureId']}</td></tr>
-	<tr><td class="cell-left">${_(zaehlstelle)}</td>        <td>${c['attributes']['zaehlstellen_bezeichnung']}</td></tr>
-    <tr><td class="cell-left">${_('physischvirtuell')}</td>   <td>${c['attributes']['zst_physisch_virtuell'] or '-'}</td></tr>
-	<tr><td class="cell-left">${_('messstellentyp')}</td>     <td>${c['attributes']['messstellentyp'] or '-'}</td></tr>
-	<tr><td class="cell-left">${_('intern')}</td>             <td><a href="http://doc.vde.admin.ch/out/intern/${c['featureId'] or '-'}.html" target="_blank">${_('linkzurbeschreibung')}</a></td></tr>
+    <tr><td class="cell-left">${_(nummer)}</td>                 <td>${c['featureId']}</td></tr>
+    <tr><td class="cell-left">${_(zaehlstelle)}</td>            <td>${c['attributes']['zaehlstellen_bezeichnung']}</td></tr>
+    <tr><td class="cell-left">${_('physischvirtuell')}</td>     <td>${c['attributes']['zst_physisch_virtuell'] or '-'}</td></tr>
+    <tr><td class="cell-left">${_('messstellentyp')}</td>       <td>${c['attributes']['messstellentyp'] or '-'}</td></tr>
 </%def>
```
Comment (oterral) [inline chsdi/templates/htmlpopup/verkehrszaehlstellen.mako]: no closing <table> tag at the end ?

**Pull request merged**